src/DIRAC/ConfigurationSystem/Client/PathFinder.py

```diff
@@ -297,3 +310,9 @@ def getGatewayURLs(system="", service=None):
         return False
     gateways = List.randomize(List.fromChar(gateways, ","))
     return [checkComponentURL(u, system, service) for u in gateways if u] if system and service else gateways
+
+
+def getDisabledDiracxVOs() -> list[str]:
+    """Get the list of VOs for which DiracX is enabled"""
+    vos = gConfigurationData.extractOptionFromCFG("/DiracX/DisabledVOs")
+    return List.fromChar(vos or "", ",")
```

Comment:
Please explain why not simply

```
return List.fromChar(vos or "")
```

Reply:
Because this was what was used in `getGatewayURLs` a few lines above and I didn't bother looking up how exactly `List.fromChar` is defined.

Reply:
IT does also the internal whitespace stripping